### ISSUE TITLE
Compiled version

### ISSUE THREAD

**Issue by NightScript370** [smealum/dscraft#1]
Is there any compiled version found on the internet?

**Comment by Caboosium** [comment]
I tried building for you but creeper_bin.h is missing.

**Comment by DaMatrix** [comment]
not sure if anyone even still pays attention to this repo, but creeper_bin.h isn't anywhere for me either.

commenting out the like with creeper_bin.h lets it build all the way, but there's linking errors (a lot of them).

@smealum any tips? i know you don't actively work on this anymore, but it'd be nice for people to be able to continue it (i know i'd like to, and wouldn't enjoy starting from scratch)

**Comment by larspelz** [comment]
I've got a compiled version. Can it help you fix the missing header file?

**Comment by DaMatrix** [comment]
i don't think so, as there's a LOT of other, seemingly unrelated errors when compiling

**Comment by smealum** [comment]
unfortunately there's a lot of reasons why things would be failing, one of which is that this relied on a custom version of libnds/libfat with hooks into the latter. i don't currently know where the code for that is and i'm swamped with work for the next month or so. i might be able to take a look later on if i'm reminded but there's no guarantee that any of that code still even exists

**Comment by smealum** [comment]
beyond that - the code for dscraft is complete trash and i would not recommend trying to extend it

**Comment by DaMatrix** [comment]
alrighty, off to making my own version it is. i'd had some pretty big changed planned anyway :P

**Comment by Steve-Tech** [comment]
Hey, sorry if this is a bit late but the download links still work if you look at the cached version of the website through archive.org, [here's a link](https://web.archive.org/web/20160818124931/http://smealum.net/dscraft/)

**Comment by oliver2999** [comment]
can someone give me dscraft for emulator no$gba

**Comment by ahino6942** [comment]
> i don't think so, as there's a LOT of other, seemingly unrelated errors when compiling

can i help with it?
